nautobot/core/tests/integration/test_filters.py

```diff
@@ -15,6 +18,20 @@ def setUp(self):
         self.login(self.user.username, self.password)
         factory.RegionFactory.create_batch(15, has_parent=False)
         factory.SiteFactory.create_batch(15, has_tenant=False)
+        # set test user to admin
+        self.user.is_superuser = True
+        self.user.save()
+
+        self.cf_text_field_name = "text_field"
+        self.cf_integer_field_name = "integer_field"
+        self.cf_select_field_name = "select_field"
```

Comment:
Mind adding a # TODO comment here about changing these from `name` to `slug` in #824?